portal-web/docroot/html/taglib/ui/input_time/page.jsp

```diff
@@ -61,14 +56,7 @@ String nameId = namespace + HtmlUtil.getAUICompatibleId(name);
 
 Calendar calendar = CalendarFactoryUtil.getCalendar(1970, 0, 1, hourOfDayValue, minuteValue, 0, 0, timeZone);
 
-String simpleDateFormatPattern = _SIMPLE_DATE_FORMAT_PATTERN_ISO;
-
-if (BrowserSnifferUtil.isMobile(request)) {
-	simpleDateFormatPattern = _SIMPLE_DATE_FORMAT_PATTERN_HTML5;
-}
-else if (amPm) {
-	simpleDateFormatPattern = _SIMPLE_DATE_FORMAT_PATTERN;
-}
+String simpleDateFormatPattern = _SIMPLE_DATE_FORMAT_PATTERN_HTML5;
```

Comment:
We can place this inline, no need for a variable.

But, is is safe to ignore `amPm`? Does vanilla time input never show it? If no, is that acceptable? Or is it applied in some other way, not through `_SIMPLE_DATE_FORMAT_PATTERN`?

Reply:
Depending on your operating system configuration, the input will show am/pm or not. For example, if your system time is 12 hours, it will display am or pm. If your system time is 24 hours, it will not show it. Marcos put an example here https://github.com/liferay-frontend/liferay-portal/pull/3284#issuecomment-1525714682.